packages/client/e2e/blotter.spec.ts

```diff
@@ -55,8 +55,8 @@ test.describe("Trade Blotter", () => {
 
   // eslint-disable-next-line no-empty-pattern
   test.afterAll(async ({}, testInfo) => {
-    if (testInfo.project.name === OPENFIN_PROJECT_NAME) {
-      await blotterPage.getByTestId("filter-button").click()
+    if (await blotterPage.isVisible("[data-testid='filter-button']")) {
```

Comment:
this `afterAll` is used to clean last test state where it add a filter. I made it conditional as the test framework execute the `afterAll` also upon test failure so the filter button may not be present in this case causing false negative.  Also made it executed for both openfin and web

From playwright documentation
https://playwright.dev/docs/api/class-test#test-after-all-1

> Details
>
> Note that worker process is restarted on test failures, and afterAll hook runs again in the new worker. Learn more about [workers and failures](https://playwright.dev/docs/test-retries).